samples/captioning/multi-track-captions.html

```diff
@@ -85,6 +152,35 @@
                     </div>
                 </div>
             </div>
+            <div class="settings">
+                <div>
+                    <button onclick="play()">play</button>
+                </div>
+                <div>
+                    <label>textDefaultEnabled</label>
+                    <input type="checkbox" id="textDefaultEnabled" onchange="setTextDefaultEnabled()"/> <span id="textDefaultEnabledValue">undefined</span>
+                </div>
+                <div>
+                    <label>initial settings Language/Role </label>
+                    <select onchange="setLang()" id="textSettings">
+                        <option value="">-- unset initial lang and role --</option>
+                        <option value="eng-subtitle">eng - subtitles</option>
+                        <option value="eng-caption">eng - captions</option>
+                        <option value="swe">swe - subtitles</option>
+                        <option value="qbb">qbb - subtitles</option>
+                        <option value="nor">nor - subtitles</option>
+                    </select>
+                </div>
+                <div>
+                    <label>streaming.lastMediaSettingsCachingInfo.enabled</label>
+                    <input type="checkbox" id="lastMediaSettingsCachingInfoEnabled" onchange="setLastMediaSettingsCachingInfoEnabled()"/>
```

Comment:
Suggest to use similar visualization as for textDefaultEnabled. Toggle between enabled and disabled:
Either [x] Enabled or [] Enabled

Reply:
Enabled by default

Reply:
Done